### PR TITLE
Prades packaging work

### PR DESCRIPTION
**Pull request by rbourianes**
https://github.com/fl4re/openvr/pull/1 reopened on the right branch.

### CODE REVIEW

**Comment by krunal-patel-fl4re** [comment]
this looks right to me.  again accept pr and we'll have to iterate on any issues encountered with publish (my responsibility)

**Review by mbarnes-sb — APPROVED** [review]
looks good.

File: build_openvr.bat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+setlocal EnableDelayedExpansion
+
+@echo off
+
+set CMAKE_GENERATOR=Visual Studio 14 2015 Win64
+call "%VS140COMNTOOLS%..\..\VC\bin\amd64\vcvars64.bat"
+if !errorlevel! neq 0 exit /b !errorlevel!
```
Comment (mbarnes-sb) [inline build_openvr.bat:7]: consider using modern `if errorlevel 1` throughout .bat files

**Comment by mbarnes-sb** [comment]
@krunal-patel-fl4re  you have to review and click approve. I've setup publish repos branch protection as such.

**Review by krunal-patel-fl4re — APPROVED** [review]
all good

**Comment by rbourianes** [comment]
Thanks @krunal-patel-fl4re 
I suppose you're already aware but publish-openvr fails.
`publish_binary_on_mac.sh: line 8: node_modules/.bin/prades: No such file or directory`
Cf. http://jk1.starbreeze.com:7070/job/build-openvr-on-mac/9/console

**Pull request merged**